examples/systemd/node_exporter.service

```diff
@@ -4,7 +4,9 @@ Requires=node_exporter.socket
 
 [Service]
 User=node_exporter
-EnvironmentFile=/etc/sysconfig/node_exporter
+# Fallback when environment file does not exist
+Environment=OPTIONS=
```

Comment:
Why this line?

Reply:
This is such that `$OPTIONS` is always set an can be expanded below without any warnings.

Reply:
Ah that makes sense